### ISSUE TITLE
How hide labels?

### ISSUE THREAD

**Issue by FanIT** [beto-rodriguez/LiveCharts2#36]
Hello. I want to use LiveChart 2 in Avalonia project. I set `IsVisible = false` in XAxis and YAxis to hide labels, but now there is empty space.  How can i hide it? 
Thanks.
![Screen](https://user-images.githubusercontent.com/20283004/117452791-dc1a0480-af4c-11eb-89f0-04fd1de37555.PNG)

**Comment by beto-rodriguez** [comment]
`Series.DataPadding` should do the trick, docs are not ready yet, but you can try something like:

``` c#
new LineSeries<int> 
{ 
    Values = new []{ 1, 5, 4, 2, 7, 4, 6, 8 },
    DataPadding = new System.Drawing.PointF(0, 1)
}
```

for both `X` and `Y` axes: `0` means nothing, `1` means the `Axis tick`, an `Axis tick` is the distance between the labels in the axis (even if there are not visible labels, the tick is calculated).

**Comment by FanIT** [comment]
@beto-rodriguez  thank you. It's works great. And thank for continuing the project.

**Comment by FanIT** [comment]
@beto-rodriguez  
I would wantn't to open a new issue, but i have a question.
I'm using CartesianChart in Avalonia. I set Background="{x:Null}" for CartesianChart to achieve a background of my window, but the CartesianChart is ignoring backround of the window and becomes completely transparent. I need to get the translucent, black background.
```
<Window xmlns="https://github.com/avaloniaui"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        xmlns:lvc="using:LiveChartsCore.SkiaSharpView.Avalonia"
        xmlns:cm="using:AvaloniaTest"
        mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
        x:Class="AvaloniaTest.MainWindow" Title="AvaloniaTest" TransparencyLevelHint="AcrylicBlur" Background="{x:Null}">
    <Window.DataContext>
        <cm:ChartModel />
    </Window.DataContext>
    <Grid>
        <Rectangle Fill="Black" Stroke="Black" Opacity="0.7" />
        <lvc:CartesianChart Series="{Binding Series}" XAxes="{Binding YAxis}"
                            YAxes="{Binding XAxis}" IsHitTestVisible="False"
                            Background="{x:Null}">
      </lvc:CartesianChart>
    </Grid>
</Window>
```
Without CartesianChart
![Screen1](https://user-images.githubusercontent.com/20283004/117569186-1f02e600-b0cd-11eb-978b-d8b520174978.PNG)
With CartesianChart
![Screen2](https://user-images.githubusercontent.com/20283004/117569097-97b57280-b0cc-11eb-88c9-6eabbfe8f880.PNG)

**Comment by beto-rodriguez** [comment]
I have also seen this issue, I am not sure yet, but I think this issue of of Avalonia.

The problem is when the chart clears the canvas with a transparent color, Avalonia has not drawn anything behind our chart control, then we get this result.

**Comment by FanIT** [comment]
Thanks for answered. I solved the problem by setting a color with alpha-canal.
```
CartesianChart cartesianChart = this.FindControl<CartesianChart>("CartesianChart");
cartesianChart.Background = new SolidColorBrush(new Color(128,0, 0, 0));
```

**Comment by FanIT** [comment]
@beto-rodriguez,  sorry. I have one more question. 
I'm using LiveChart 2 for real time chart and watching a memory leak and increasing cpu load.
I'm add a new ObservablePoint and remove a first in a ObservableCollection every second.
Should i somehow clearing CartesianChart or do some other an actions for to free the memory?

**Comment by beto-rodriguez** [comment]
it should be handled automatically by tue library, let me try to reproduce

**Comment by FanIT** [comment]
I builded LiveChartsCore.SkiaSharpView.dll and LiveChartsCore.SkiaSharpView.Avalonia.dll from fresh source code and tested with JetBrains dotMemory.
I still watching a memory leak.
Total used:
Start - 51.4 mb
10 minutes - 80.9 mb
20 minutes - 114.3 mb
30 minutes - 150.8 mb
40 minutes - 180.5 mb
50 minutes - 215.6 mb
Maybe i'm doing something wrong?
I can provide my test code or any required data.
My chart model.
```
class ChartModel
    {
        private Random random;
        private int index = 0;

        public ObservableCollection<ObservablePoint> observableValues { get; set; }

        public ChartModel()
        {
            random = new Random();

            observableValues = new();

            for (byte b = 0; b < 50; b++)
            {
                observableValues.Add(new ObservablePoint(index++, b));
            }

            Series = new ObservableCollection<ISeries>
            {
                new LineSeries<ObservablePoint>
                {
                    Values = observableValues,
                    Fill = new SolidColorPaintTask(SKColors.Blue),
                    Stroke = null,
                    GeometryFill = null,
                    GeometryStroke = null,
                    DataPadding = new PointF(0, 1)
                }
            };
        }

        public void Add()
        {
            observableValues.RemoveAt(0);
            observableValues.Add(new ObservablePoint(index++, random.NextDouble()));
        }

        public ObservableCollection<ISeries> Series { get; set; }

        public List<Axis> XAxis { get; set; } = new List<Axis>
        {
            new Axis
            {
                IsVisible = false
            }
        };

        public List<Axis> YAxis { get; set; } = new List<Axis>
        {
            new Axis
            {
                IsVisible = false
            }
        };
    }
```

**Comment by beto-rodriguez** [comment]
OK I'll keep investigating.

**Comment by beto-rodriguez** [comment]
@FanIT How are you calling the `Add` method, I just tested the `AvaloniaSample` project, in the lines `AutomaticUpdates` sample, I see no memory leak after 30 mins running.

I will let it more time running, and will document my results here, but at first instance I think that the issue is gone.

**Comment by beto-rodriguez** [comment]
50 mins running, no memory leak found, tested with dot memory 2021.1

![image](https://user-images.githubusercontent.com/10853349/118053887-f425c580-b34a-11eb-9377-22d2da60daa7.png)

**Comment by FanIT** [comment]
Many thanks you and sorry. Probably the problem was in Avalonia. I updated to Avalonia 0.10.4 and no longer see the memory leak.
I'm calling Add in DispatcherTimer of Avalonia.Threading a every second.
```
Avalonia.Threading.DispatcherTimer timer = new DispatcherTimer();
            timer.Interval = new TimeSpan(0, 0, 1);
            timer.Tick += (object? sender, EventArgs e) => { chart.Add(); };
            timer.Start();
```
I tested 1 hour and every 10 minutes doing snapshot.
![dotMemory](https://user-images.githubusercontent.com/20283004/118105636-4e736480-b3e5-11eb-8041-401ff73d046c.png)
